docs/install.asciidoc

```diff
@@ -26,6 +26,20 @@ Debezium executes nightly builds and deployments into the Central snapshot repos
 
 To use a connector to produce change events for a particular source server/cluster, simply create a configuration file for the link:/docs/connectors/mysql/#configuration[MySQL Connector] or link:/docs/connectors/mongodb/#configuration[MongoDB Connector], and use the link:http://docs.confluent.io/{confluent-platform-version}/connect/userguide.html#rest-interface[Kafka Connect REST API] to add that connector configuration to your Kafka Connect cluster. When the connector starts, it will connect to the source and produce events for each inserted, updated, and deleted row or document. See the Debezium documentation for the link:/docs/connectors/mysql/[MySQL Connector] and link:/docs/connectors/mongodb/[MongoDB Connector].
 
+== Configuring Debezium Topics
+Debezium uses (either via Kafka Connect or directly) multiple topics for storing data. The topics has to be either created by an administrator or by Kafka itself by enabling auto-creation for topics. There are certain limitations and recommendations which apply to topics
+
+* Database history topic (for MySQL connector)
+** Infinite (or very long) retention (no compaction!)
+** Replication factor at least 3 for production
+** Single partition
+* Other topics
+** Compaction enabled
+** Replicated in production
+** Single partition
+*** You can relax the single partition rule but your application must handle out-of-order events for different rows in database (events for a single row are still totally ordered).
```

Comment:
if multiple petitions are used, Kafka will determine the partition by hashing the key by default. Other partition strategies require using SMTs to set the partition number for each record.

Reply:
Done